src/classes/HiFiAudioAPIData.ts

```diff
@@ -488,8 +488,15 @@ export class ReceivedHiFiAudioAPIData extends HiFiAudioAPIData {
      * ✔ The server sends `volumeDecibels` data to all clients connected to a server during "peer updates".
      */
     volumeDecibels: number;
+
+    /**
+     * Indicates that the peer is providing stereo audio.
+     *
+     * The server sends `isStereo` data to all clients connected to a server during "peer updates".
+     */
+    isStereo: boolean;
     
-    constructor(params: { providedUserID?: string, hashedVisitID?: string, volumeDecibels?: number, position?: Point3D, orientationQuat?: OrientationQuat3D, hiFiGain?: number } = {}) {
+    constructor(params: { providedUserID?: string, hashedVisitID?: string, volumeDecibels?: number, position?: Point3D, orientationQuat?: OrientationQuat3D, hiFiGain?: number, isStereo?:boolean } = {}) {
```

Comment:
please fix spacing, i.e.:
```suggestion
    constructor(params: { providedUserID?: string, hashedVisitID?: string, volumeDecibels?: number, position?: Point3D, orientationQuat?: OrientationQuat3D, hiFiGain?: number, isStereo?: boolean } = {}) {
```

and then add `this.isStereo = params.isStereo;` to the bottom of the function.

you'll also have to add `isStereo: boolean;` with documentation to the `ReceivedHiFiAudioAPIData` class definition.